data/World/Spirit Temple.json

```diff
@@ -166,7 +198,11 @@
             "Spirit Temple Statue Room Hand Chest": "can_play(Zeldas_Lullaby)",
             "Spirit Temple Statue Room Northeast Chest": "
                 can_play(Zeldas_Lullaby) and
-                (Hookshot or Hover_Boots or logic_spirit_lobby_jump)"
+                (Hookshot or Hover_Boots or logic_spirit_lobby_jump)",
+            
+            "Spirit Temple Adult Climb Flying Pot 1" : "True",
+            "Spirit Temple Adult Climb Flying Pot 2" : "True",
```

Comment:
When you enter this room from the upper door, and you're heading down to the pots, you have to take care to walk around the edges of the room. You could aggro the pots from above, which causes them to rise up through the floor and try to attack you in the top half of the room. It's very easy to miss that this is happening. Obviously you could just reload using the lower door, but what if you don't have a key for it?

There's not really anything to be done here. I think the logic is fine as-is, but I wanted to call attention to a rather janky interaction.